### DISCUSSION TITLE
waline的登录界面下方的github登陆

### DISCUSSION THREAD

**Discussion by zxmacc** [walinejs/waline#187] [Q&A] [answered]
waline的登录界面下方的github登陆要自己配置吗，我看了几位别的博主都有，我的没有。
![Screenshot_2021-03-27-12-33-46-651_com baidu searchbox](https://user-images.githubusercontent.com/71143103/112709973-be587a80-8e8b-11eb-943e-976430848a7d.jpg)
![Screenshot_2021-03-27-12-33-16-865_com baidu searchbox](https://user-images.githubusercontent.com/71143103/112709974-c0223e00-8e8b-11eb-9aa3-356cbfbb81ec.jpg)

**Comment by zxmacc** [comment]
版本都显示是0.14.8

**Comment by lizheming** [comment, marked as answer]
https://waline.js.org/server/socials.html#github
